CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 14.0.0
+
+#### Breaking changes - [#525](https://github.com/openfisca/openfisca-core/pull/525)
+
+* Rename Variable attribute `url` to `reference`
+  - As previously, this attribute can be a string, or a list of strings.
+  - Variable attribute `law_reference` is removed.
+  - `TaxBenefitSystem.reference` renamed to `TaxBenefitSystem.reference_tbs`.
+  - Parameter `reference` of `AbstractScenario.new_simulation()` renamed to `use_reference_tbs`.
```

Comment:
The two `reference` options you're renaming to `use_reference_tbs` have always sounded weird to me: `reform.legislation_at(x, reference=True)` returns the legislation of the tax benefit system the reform was built on 😕 .
I don't know if it's relevant to rename something we may just want to depreciate one of these days 🤔 .